### ISSUE TITLE
List not rendering as values

### ISSUE THREAD

**Issue by jasonvogel** [postsharp/PostSharp.Community.ToString#2]
**Results**

> {Task; Id:7358675e-6a50-456a-85bb-3c44470c4239,Description:Task 1,DueDate:01/01/2020,Tags:null}
> {Task; Id:6e25336d-6a4c-4108-975d-92181448ee3e,Description:Task 2,DueDate:02/02/2020,Tags:System.Collections.Generic.List`1[TaskFighter.Tag]}
> {Task; Id:ff379661-825c-4ff3-9c28-11aa3c09ece8,Description:Task 3,DueDate:03/03/2020,Tags:null}
> {Task; Id:e8047132-7c67-48c1-8e14-f5db83593694,Description:Task 4,DueDate:04/04/2020,Tags:System.Collections.Generic.List`1[TaskFighter.Tag]}

So why am I seeing... "Tags:_System.Collections.Generic.List`1[TaskFighter.Tag]_"?

I expected to see "Tags:Default" and "Tags:Default4"

**Environment**
 - MacOS: 10.15.5
 - dotnet --version: 3.1.301
 - Visual Studio Community: 8.6.5 (build 23)
 - PostSharp.Community.ToString: 1.1.0

**Code** 
```	[ToString]
	public class TaskList : ITasks
	{
		public TaskList()
		{
		}

		public List<Task> Tasks { get; set; }

		public List<Task> GetData()
		{
			return Tasks;
		}

		public ITasks FilterByTag(Tag tag)
		{
	Tasks = Tasks.Where(s => s.Tags != null &&
								s.Tags.Any(t => t.Label == tag.Label)).ToList();

			return this;
		}

		public ITasks LoadTasks()
		{
			Tasks = new List<Task>()
			{
				new Task { Id = Guid.NewGuid(), Description = "Task 1", DueDate = "01/01/2020" },
				new Task { Id = Guid.NewGuid(), Description = "Task 2", DueDate = "02/02/2020" },
				new Task { Id = Guid.NewGuid(), Description = "Task 3", DueDate = "03/03/2020" },
				new Task { Id = Guid.NewGuid(), Description = "Task 4", DueDate = "04/04/2020" }
			};

			Tag tag = new Tag { Label = "Default" };

			Task task = Tasks[1];
			task.AddTag(tag);

			Tag tag4 = new Tag { Label = "Default4" };
			task = Tasks.Last();
			task.AddTag(tag4);

			return this;
		}
	}
}
var tasks = new TaskList().LoadTasks().GetData();
foreach (var task in tasks)
{
Console.WriteLine(task);
}

[ToString]
public class Task
{
	public Guid Id;
	public string Description;
	public string DueDate;

	public List<Tag> Tags { get; set; }

	public void AddTag(Tag tag)
	{
		if (null == Tags)
			Tags = new List<Tag>();

		Tags.Add(tag);
	}
}

[ToString]
public class Tag
{
	public string Label;
}

public interface ITasks
{
	ITasks LoadTasks();
	ITasks FilterByTag(Tag tag);

	List<Task> GetData();
	List<Task> Tasks { get; set; }
}
```

**Comment by Soothsilver** [comment]
I think what you would have expected to see is something like `Tags:[{Tag, Label:Default}]` but I'm not sure what the behavior should be for longer lists. At some point, ToString'ing the entire collection seems like a bad idea but it's still probably better than printing the type of the collection.

**Comment by jasonvogel** [comment]
Correct or _Tags: [Label:Default]_.
Yeah, the type of the collection never seems to have much value to me.  Most of the time, my collections are generally small (from a testing standpoint).

**Comment by Soothsilver** [comment]
Fixed in #3 in 1.2.0, published on NuGet. 

I added that we print collections and arrays as `[1, 2, 3, 4, ...]` (the first four elements of the enumerable).

**Comment by jasonvogel** [comment]
Confirmed, 1.2.0 is working as expected.  Thanks!
